seatunnel-core/seatunnel-starter/src/main/java/org/apache/seatunnel/core/starter/seatunnel/command/ClientExecuteCommand.java

```diff
@@ -281,10 +281,16 @@ private HazelcastInstance createServerInLocal(
         // set the default async executor for Hazelcast InvocationFuture
         ConcurrencyUtil.setDefaultAsyncExecutor(CompletableFuture.EXECUTOR);
 
-        return HazelcastInstanceFactory.newHazelcastInstance(
-                seaTunnelConfig.getHazelcastConfig(),
-                Thread.currentThread().getName(),
-                new SeaTunnelNodeContext(seaTunnelConfig));
+        try {
+            return HazelcastInstanceFactory.newHazelcastInstance(
+                    seaTunnelConfig.getHazelcastConfig(),
+                    Thread.currentThread().getName(),
+                    new SeaTunnelNodeContext(seaTunnelConfig));
```

Comment:
create hazelcast instance is one of potential crisis. So we should handle it in https://github.com/apache/seatunnel/blob/0b1122ceda60b4e94a9c2665e33f0d893644b22d/seatunnel-core/seatunnel-core-starter/src/main/java/org/apache/seatunnel/core/starter/SeaTunnel.java#L40 to catch all exist problem.